msdsn/conn_str.go

```diff
@@ -72,6 +74,15 @@ type Config struct {
 	ConnTimeout time.Duration // Use context for timeouts.
 	KeepAlive   time.Duration // Leave at default.
 	PacketSize  uint16
+
+	// Kerberos authentication fields
+
+	Krb5ConfFile      string
+	KrbCache          string
+	Realm             string
```

Comment:
What purpose does "realm" has. How does someone know what to put here?

Reply:
Have added description for the same in comments for the kerberos fields